tools/cli/go-whisk/whisk/client.go

```diff
@@ -162,11 +140,43 @@ func NewClient(httpClient *http.Client, config *Config) (*Client, error) {
     return c, nil
 }
 
+func (c *Client) LoadX509KeyPair() error {
+    tlsConfig := &tls.Config {
+        InsecureSkipVerify: c.Config.Insecure,
+    }
+
+    if c.Config.Cert != "" && c.Config.Key != "" {
```

Comment:
should a warning/error be returned if the cert is partially configured - i.e. a key is specified, but not the cert or a cert is specified but not the key?  currently, this incorrect configuration is silently ignored.

Reply:
done.